### ISSUE TITLE
ESC not working to exit full screen chrome and firefox

### ISSUE THREAD

**Issue by developersteve** [billziss-gh/EnergyBar#48]
No idea on this one but Chrome and Firefox seems to have an issue with ESC not working to exit full screen mode (works with safari). 

Also trying to use fn + esc doesnt seem to work either so unsure if this is a energybar bug or a touchbar issue but keen to hear if anyone else has the same issue.

**Comment by billziss-gh** [comment]
If <kbd>fn</kbd>+<kbd>esc</kbd> (which completely bypasses EnergyBar) does not work, then it must be a problem with those 2 pieces of software.

Does the problem still happen when EnergyBar is not running?

**Comment by orangecoding** [comment]
Esc. is not working for me either. fn+esc works.

**Comment by billziss-gh** [comment]
Have you made sure that EnergyBar is enabled under Privacy > Accessibility in your System Preferences?

**Comment by developersteve** [comment]
@billziss-gh Yep tried uninstalling and still couldnt do it although possibly it was still running (had to do a restart but not sure if there was a process that still started). 

@orangecoding yeah noticed that did work, still not sure if its a core osx quirk cause the default bar also doesnt work. Did read on a few forums that FF and Chrome had changed the ESC api recently so not sure if thats related too.

**Comment by orangecoding** [comment]
> Have you made sure that EnergyBar is enabled under Privacy > Accessibility in your System Preferences?
Yes, I checked everything twice.

**Comment by ma-jahn** [comment]
i have just installed Energybar and noticed that ESC is also not working for me. fn+esc does work.

Also the "play / pause" button doesn't seem to work (doesn't start or pause any song in itunes / spotify). If required i'll create a separate issue for it.
